src/main.rs

```diff
@@ -112,6 +112,11 @@ fn main() {
                             .long("right")
                             .help("Print the right prompt (instead of the standard left prompt)"),
                     )
+                    .arg(
+                        Arg::with_name("continuation")
+                            .long("continuation")
+                            .help("Print the continuation prompt (instead of the standard prompt)"),
```

Comment:
I think the language should match `--right` and the new argument should be marked as incompatible with `--right`.

```suggestion
                            .help("Print the continuation prompt (instead of the standard left prompt)")
                            .conflicts_with("right"),
```